functions/src/api/sessions/index.ts

```diff
@@ -84,12 +85,15 @@ sessionsRouter.post('/', validator({body: CreateSessionSchema}), async ctx => {
     timestamp: Timestamp.now(),
   };
 
+  const link = await createDynamicLink(`sessions/${data.id}`);
```

Comment:
I imagine that would be the invitecode instead of the id for private sessions in the future? 🤔

Reply:
actually both would work since we could fetch the data from either 🙌

Reply:
Yeah I was thinking the same. That for private sessions we could have another pattern. E.g. `sessions/:sessionId/invitation/:invitationCode` or similar.

Reply:
This is where you later setup the routing in the client:

https://github.com/29ki/29k/blob/08892f96c351a0e71dc1854c441d2b1c65f4b977/client/src/lib/navigation/linking.ts#L10-L19